pkg/kubelet/cm/container_manager.go

```diff
@@ -118,6 +119,12 @@ type ContainerManager interface {
 
 	// UpdateAllocatedDevices frees any Devices that are bound to terminated pods.
 	UpdateAllocatedDevices()
+
+	// GetAllocatableDevices returns information about all the devices known to the manager
+	GetAllocatableDevices() map[string]map[string]pluginapi.Device
+
+	// GetAllocatableCPUs returns the assignable (not allocated) CPUs
+	GetAllocatableCPUs() []int64
```

Comment:
Is there a reason not to have this return a `cpuset.CpuSet()`?  Using tzhis tye makes it more flexible if we need to perform any operations on the returned CPUs in the future.

Likewise, I saw a discussion on wrapping `map[string]map[string]pluginapi.Device` in its own type (which I agree with), which would then make this symmetzric with that.

Reply:
There is no good reason that I can recall, so I'll change accordingly.
Same goes for wrapping `map[string]map[string]pluginapi.Device` in a new type, which I believe can sit inside the devicemanager package, right?

Reply:
Yes. I would put it inside `pod_devices.go`.

Reply:
Perfect. On it.